csharp/device/Microsoft.Azure.Devices.Client/Message.cs

```diff
@@ -144,7 +178,11 @@ public string To
         {
             get
             {
+#if NETMF
+                return (string)(this.GetSystemProperty(MessageSystemPropertyNames.To) ?? string.Empty);
```

Comment:
Please change it. I did replace all the others as we've discussed. I must have missed this one...